### DISCUSSION TITLE
.env-file and directories are not writable

### DISCUSSION THREAD

**Discussion by b3nd1k** [Kovah/LinkAce#397] [Support]
For the past few days I've been trying to get LinkAce up and running on my Ubuntu-server. I've it installed using docker compose, but LinkAce keeps telling me that my .env-file, /storage and /storage/logs are not writable. 

<img width="502" alt="Screenshot 2022-02-12 at 17 04 43" src="https://user-images.githubusercontent.com/96381857/153718691-b4194769-281c-455e-9b94-160193d89029.png">

I've attached my stack. Does anyone know why the paths are not writable and how to fix it? 

`version: "3"
services:
  linkace:
    hostname: linkace
    image: linkace/linkace:simple
    container_name: linkace
    networks:
      - milkyway
    ports:
      - 8585:80/tcp
    restart: unless-stopped
    volumes:
      - /mnt/volume/appdata/linkace/.env:/app/.env
      - /mnt/volume/appdata/linkace/storage/logs:/app/storage/logs
    working_dir: /app
    restart: unless-stopped
networks:
  milkyway:
    external: true`

**Comment by martadinata666** [comment]
```
If you are unsure if the .env file is writable inside Docker, please make it writable for anybody (-rw-rw-rw- or 666). You can switch back to make it read only after the setup.
```
https://www.linkace.org/docs/v1/setup/setup-with-docker/simple/

So it clear enough, and we skipped things. 🤦🏼

**Reply by Kovah** [reply]
Did that work for you?

**Reply by martadinata666** [reply]
As @b3nd1k deploy via portainer, i cant really tell how it manage permission, before this he also had a thread and seems instead using `:simple`  tag he use `:latest` tag, common way to deploy. We get it, but stuck at this perm issue, as i deploy manually i got my personal docker php+nginx, so i know which permission to use. And only few hours ago i tried your docs. Well i can assure it work, 

```
drwxrwxr-x 1 1000 1000  266 Feb 13 11:39 db/
-rw-rw-rw- 1 1000 1000 3251 Feb 13 11:30 env
drw-rw-rw- 1 1000 1000   88 Feb 13 10:59 logs/ 

```
So doesn't matter who own it set `666` will make everyone can read and write it, not really recommended, but it is what it is 👍🏼

**Reply by b3nd1k** [reply]
> ```
> If you are unsure if the .env file is writable inside Docker, please make it writable for anybody (-rw-rw-rw- or 666). You can switch back to make it read only after the setup.
> ```

I'm sorry, but how do I make it writeable for anybody? Which command is that?

**Reply by martadinata666** [reply]
Like previous command disccussed.
```
chmod 666 /the/path/to/env
```

**Reply by b3nd1k** [reply]
Using CHMOD 666 on both paths worked and the paths now show as writeable and I was able to finish the setup, thank you! I appreciate y'alls patience.
